packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -62,6 +62,11 @@ export const ConnectedRichTextEditor = (props: ConnectedProps) => {
     props.isToolbarHidden && styles.hiddenToolbar
   );
 
+  useEffect(() => {
+    getPlateActions(id).enabled(true);
+    return () => getPlateActions(id).enabled(false);
+  }, [id]);
```

Comment:
We will probably forget why we have this magic here. Let's add a comment to remind us.